### ISSUE TITLE
EU Flag missing?

### ISSUE THREAD

**Issue by CKKwan** [emcrisostomo/flags#1]
I know EU is not a nation, but it will be good if the flag is included. Thx

**Comment by Foorack** [comment]
Yup 🇪🇺 is an emoji. Most platforms implement [Emoji flag sequences](https://en.wikipedia.org/wiki/Regional_indicator_symbol#Emoji_flag_sequences) which are 256 flags in total (249 ISO-3166-1-alpha-2 codes, 6 exceptional reservations, and 1 usera-ssigned temporary code for Kosovo).

<img width="616" height="236" alt="Image" src="https://github.com/user-attachments/assets/80b8c9b3-60c4-4d36-86b4-8028ab101890" />
